throttler.go

```diff
@@ -148,12 +148,18 @@ func (t *LagThrottler) Run(ctx context.Context) error {
 }
 
 func (t *LagThrottler) updateLag(ctx context.Context) error {
-	var newLag int
+	var newLag sql.NullInt64
 	err := t.DB.QueryRowContext(ctx, t.config.Query).Scan(&newLag)
+	if err == sql.ErrNoRows {
+		return nil
+	}
 	if err != nil {
 		return err
 	}
+	if !newLag.Valid {
```

Comment:
"Valid" means "not null"?

Reply:
yup!

Reply:
newLag is now sql.NullInt64 which has Valid as a field.